dotnetv3/Bedrock-runtime/Models/MetaLlama/Llama2_InvokeModel/InvokeModel.csx

```diff
@@ -0,0 +1,60 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// snippet-start:[BedrockRuntime.dotnetv3.InvokeModel_MetaLlama2]
+// Use the native inference API to send a text message to Meta Llama 2.
+
+using Amazon;
+using Amazon.BedrockRuntime;
+using Amazon.BedrockRuntime.Model;
+using System;
+using System.IO;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+
+// Create a Bedrock Runtime client in the AWS Region you want to use.
+var client = new AmazonBedrockRuntimeClient(RegionEndpoint.USEast1);
+
+// Set the model ID, e.g., Llama 2 Chat 13B.
+var modelId = "meta.llama2-13b-chat-v1";
+
+// Define the prompt for the model.
+var prompt = "Describe the purpose of a 'hello world' program in one line.";
+
+// Embed the prompt in Llama 2's instruction format.
+var formattedPrompt = $"<s>[INST] {prompt} [/INST]";
+
+//Format the request payload using the model's native structure.
+var nativeRequest = JsonSerializer.Serialize(new
+{
+    prompt = formattedPrompt,
+    max_gen_len = 512,
+    temperature = 0.5
+});
+
+// Create a request with the model ID, the user message, and an inference configuration.
+var request = new InvokeModelRequest()
+{
+    ModelId = modelId,
+    Body = new MemoryStream(System.Text.Encoding.UTF8.GetBytes(nativeRequest)),
+    ContentType = "application/json"
```

Comment:
Should there be an inference configuration here as the comment suggests?

Reply:
Clarified, see above